### ISSUE TITLE
Edit KVM guest(VM)  configurations (USB port, sound card, etc.) with ganeti

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#396]
Originally reported of Google Code with ID 340.

```
https://groups.google.com/forum/?fromgroups=#!topic/ganeti/4B4lV1OvsXs

 
Similar to the issue 278: Implement USB pass-through in KVM ?
http://code.google.com/p/ganeti/issues/detail?id=278&q=usb&colspec=ID%20Type%20Component%20Status%20Priority%20Stars%20Opened%20Modified%20Milestone%20Owner%20Summary
```
Originally added on 2013-01-03 13:41:23 +0000 UTC.

Mentioned Issue with GoogleCode ID 278 has been migrated to GitHub with ID #334.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I personally do not need sound card for servers, but it would be good to enable usb port for gsm modems and usb keys that some programs use. 

I'm not sure about command line flags. 
```
Originally added on 2013-01-03 14:22:13 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-01-11 16:45:02 +0000 UTC.

Added Labels: Hypervisor-KVM
Changed State: Accepted
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been fixed in 2.7, with various patches.
```
Originally added on 2013-02-26 15:59:11 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
